packages/mgt-components/src/components/mgt-login/mgt-login.ts

```diff
@@ -80,6 +79,8 @@ type PersonViewConfig = {
  * @cssprop --login-command-button-text-color - {Color} the color for the text of the command button.
  * @cssprop --login-command-button-background-color - {Color} the color for the background of the command button.
  * @cssprop --login-command-button-hover-background-color - {Color} the color for the background of the command button on hovering.
+ * @cssprop --login-account-item-hover-bg-color - {Color} the background color of the account item on hover.
```

Comment:
@Mnickii is "account item" here supposed to mean one of the listed accounts when you have multiple accounts in the flyout?

Reply:
Yeah. IMO I think we should remove the account-item background and hover background tokens and handle them with person tokens rather than login tokens

Reply:
I _partially_ agree with you on that. Currently, the accounts in the list are customized with `mgt-person` tokens. If we want additional customization for `mgt-login` - in this case a hover scenario on the accounts - we need to use this token to set it to whatever color. Therefore, we can still have the token but change how it works by implementing a hover selector on `mgt-login` that changes the background and `--person-background-color` token of accounts in the list. What do you think about that?

Reply:
Yeah, nice idea! This would work